lib/src/pie_canvas_core.dart

```diff
@@ -574,9 +581,13 @@ class PieCanvasCoreState extends State<PieCanvasCore>
     _attachTimer?.cancel();
 
     if (_state.active) {
-      if (_isBeyondPointerBounds(offset) || _pressedAgain) {
-        final hoveredAction = _state.hoveredAction;
+      final int? hoveredAction = _state.hoveredAction;
+
+      final bool beyondBounds = _theme.alwaysPlaceActionFromCenter
+          ? (hoveredAction != null)
```

Comment:
I'm little unsure about what troubles it could cause later...

Shouldn't affect when `alwaysPlaceActionFromCenter` is false

Reply:
Now when the pointer leaves the menu widget, it dismisses.

Uses `RenderBox.hitTest()`